tests/core/test_cascade_plan.py

```diff
@@ -0,0 +1,43 @@
+import networkx as nx
+
+from cascade.core.parameters import _ParameterHierarchy
+from cascade.core.cascade_plan import CascadePlan
+from cascade.testing_utilities import make_execution_context
+
+
+def test_create(ihme):
+    ec = make_execution_context(location_id=0, gbd_round_id=5)
+    settings = _ParameterHierarchy(
```

Comment:
Huh, cute. I never thought of using it that way.

Reply:
I get freer in unit tests, and poke at things and twist them out of expediency. Sort of sloppy, but definitely also learning. It's interesting how that works.